app/models/concerns/benchmark_indicator_action_seed.rb

```diff
@@ -68,5 +69,38 @@ def seed_influenza_actions!
         prev_display_abbreviation = display_abbreviation
       end
     end
+
+    def seed_cholera_actions!
+      cholera = Disease.cholera
+      return if disease_actions_are_present?(cholera)
+
+      warn "Seeding data for Cholera Actions..."
+
+      cholera_actions_attrs =
+          JSON.parse File.read File.join Rails.root,
+                                         "/db/seed-data/cholera_actions.json"
```

Comment:
`Rails.root` is a `Pathname`, so you can do this: `JSON.parse Rails.root.join("db/seed-data/cholera_actions.json").read`

Reply:
fixed